domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -475,6 +492,11 @@ static JsonObject getPostgreSQLClientConfig(String tenantId, String schemaName,
       throws Exception {
     // static function for easy unit testing
     JsonObject config = environmentVariables;
+
+    Object v = config.remove(Envs.DB_EXPLAIN_QUERY_THRESHOLD.name());
+    if (v instanceof Long) {
+      PostgresClient.setExplainQueryThreshold((Long) v);
+    }
```

Comment:
This should go after the `if (config == null)` block. This ensures that config is not null, and the the config can be read from the config file at `getConfigFilePath()`.